diff --git a/node_modules/sotion/dist/components/BlockRenderer.svelte b/node_modules/sotion/dist/components/BlockRenderer.svelte
index c646014..44ad62a 100644
--- a/node_modules/sotion/dist/components/BlockRenderer.svelte
+++ b/node_modules/sotion/dist/components/BlockRenderer.svelte
@@ -5,7 +5,7 @@
   import Header from "./blocks/Header.svelte";
   import SubHeader from "./blocks/SubHeader.svelte";
   import SubSubHeader from "./blocks/SubSubHeader.svelte";
-  import ToDo from "./blocks/ToDo.svelte";
+  import ToDo from "sotion/dist/components/blocks/Todo.svelte";
   import Image from "./blocks/Image.svelte";
   import Quote from "./blocks/Quote.svelte";
   import CollectionViewPage from "./blocks/CollectionViewPage.svelte";
@@ -34,12 +34,14 @@
   };
 </script>
 
-<script>;
-export let block;
-function mapNotionComponent(block) {
+<script>
+  export let block;
+  function mapNotionComponent(block) {
     const componentBlock = blockTypeComponentMap[block.type];
-    return componentBlock !== null && componentBlock !== void 0 ? componentBlock : Error;
-}
+    return componentBlock !== null && componentBlock !== void 0
+      ? componentBlock
+      : Error;
+  }
 </script>
 
 {#if block.type === "column_list"}
