diff --git a/node_modules/sotion/src/sotion.ts b/node_modules/sotion/src/sotion.ts
index 81ccc57..88e165a 100644
--- a/node_modules/sotion/src/sotion.ts
+++ b/node_modules/sotion/src/sotion.ts
@@ -34,7 +34,7 @@ class Sotion {
 
   async fetchPage(id: string) {
     throwIfNoFetcherSet();
-    await this.getScope();
+    // await this.getScope();
 
     const res: Response = await fetcher(`${apiUrl}page/${id}`);
     const rawBlocks = await res.json();
